TransferMethodRepository/Sources/TransferMethodRepository.swift

```diff
@@ -50,10 +50,14 @@ public protocol TransferMethodRepository {
     ///   - completion: the callback handler of responses from the Hyperwallet platform
     func listTransferMethod(
         _ completion: @escaping (Result<HyperwalletPageList<HyperwalletTransferMethod>?, HyperwalletErrorType>) -> Void)
+
+    /// Refreshes transfer methods
+    func refreshTransferMethods()
 }
 
 final class RemoteTransferMethodRepository: TransferMethodRepository {
     private static let deactivateNote = "Deactivating Account"
```

Comment:
Discussed with Willem on the notes. No need to set note during deactivating an account. So remove it as well